### ISSUE TITLE
Pretrained weights for mb3-small-ssd-lite backbone

### ISSUE THREAD

**Issue by AnhPC03** [qfgaohao/pytorch-ssd#142]
Could you give any link to download pretrained weights for backbone if i want to train mb3-small-ssd-lite model?
I was trained from scratch but my dataset is small and the loss was high. when i referenced, the model drawn on image not highly corrected.
Thank you.
